src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/SqlManagerController.php

```diff
@@ -175,30 +175,20 @@ public function processFormAction(Request $request)
      */
     public function createAction(Request $request)
     {
-        $requestSqlFormHandler = $this->getRequestSqlFormHandler();
-
-        $requestSqlData = [];
-
-        // handle "Export to SQL Manager" action data
-        if ($request->request->has('sql')) {
-            $requestSqlData['request_sql'] = [
-                'sql' => $request->request->get('sql'),
-                'name' => $request->request->get('name'),
-            ];
-        }
+        $sqlRequestFormDataHandler = $this->get('prestashop.core.sql_manager.form.sql_request_form_data_handler');
```

Comment:
maybe you could add some PHPDoc so that we see what type of objects/interfaces we are dealing with? (with searching in service declaration what is what)